todo/models.py

```diff
@@ -26,6 +26,6 @@ class Status(models.TextChoices):
 
 class Log(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
-    name_log = models.CharField(max_length = 200, null=True )
+    name_log = models.CharField(max_length = 200, null=True)
     action = models.CharField(max_length=100)
-    timestamp = models.DateTimeField()
\ No newline at end of file
+    timestamp = models.DateTimeField(auto_now_add = True)
```

Comment:
```suggestion
    timestamp = models.DateTimeField(auto_now_add=True)
```

ไม่ต้องมี space ระหว่าง = ครับบ `name_log` ด้วยนะ